fio_wrapper/trigger_fio.py

```diff
@@ -79,44 +79,51 @@ def _log_payload(self, directory, user, uuid, sample, fio_jobs_dict, fio_version
             del _current_log_files['clat']
         if 'disable_bw' in job_options or 'disable_bw_measurement' in job_options:
             del _current_log_files['bw']
+        #find the number of jobs either in the job options or global options
+        if 'numjobs' in job_options:
+            numjob_list = job_options['numjobs']
+        else:
+            numjob_list = fio_jobs_dict['global']['numjobs']
+            
         for log in _current_log_files.keys():
             for host in list_hosts:
-                log_file_prefix_string = 'write_' + str(log) + '_log'
-                if log in ['clat','slat']:
-                    log_file_prefix_string = 'write_lat_log'
-                try:
-                    log_file_name = str(job_options[log_file_prefix_string]) + '_' + str(log) + '.1.log.' + str(host)
-                except KeyError:
+                for numjob in numjob_list:
```

Comment:
numjob_list would contain an integer like `4` or such, but python being python would assume the type is a string, so convert it to '4' and this for loop would only iterate over the value 4, but not from [1-4] which is what we want.